ws/DiscordWebsocket.ts

```diff
@@ -42,13 +42,9 @@ export default class DiscordWebsocket {
     return this;
   }
 
-  disconnect(code?: number) {
-    if (code === 1000 || code === 1001) {
-      this.sessionID = null;
-      this._seq = null;
-    }
+  disconnect() {
```

Comment:
Wait, why get rid of the param? We still want the option of specifying a close code, and all disconnections should happen from here instead of through the ws directly